appsec/src/extension/commands_helpers.h

```diff
@@ -36,10 +36,11 @@ dd_result dd_command_proc_resp_verd_span_data(mpack_node_t root,
     void *unspecnull ctx /* actually struct req_info* nonnull */);
 
 /* Common helpers */
+void dd_command_process_settings(mpack_node_t root);
```

Comment:
Is this called from outside of this module? Otherwise it could be private to it

Reply:
right, fixed it :+1: